src/test/server/database/test_database.py

```diff
@@ -49,11 +50,22 @@ def setUp(self) -> None:
         )
         self.job2.uid = "asv"
         self.work_machine = WorkMachine("machi1", WorkMachineState.ONLINE,
-                                        WorkMachineResources(ResourceAllocation(12, 32, 12)))
+                                        WorkMachineResources(ResourceAllocation(12, 32, 12)),
+                                        SSHConfig(
+                                            hostname="om", username="ux", password="45", key_filename="rsa",
```

Comment:
Can the database work with some of those set to None? I think we do not have all of the fields, all of the time.

Reply:
the database for sure can, I just copy pasted them this way, also it can always store None, the problem was with using to_dict.

Reply:
I think it would still be nice to have it included in some tests. Maybe you could set a few of these here to None?